src/translate-once.js

```diff
@@ -66,7 +66,7 @@
         // queue the translation
         $translate(translationKey, translateValues).then(function (translation) {
           // update the element with the translation
-          element.html(translation);
+          element.text(translation);
```

Comment:
Translations allow for HTML entities, this would be a breaking change and would lack support from `angular-translate`. Further discussion will be had in #20.